source/apiVolontaria/volunteer/admin.py

```diff
@@ -14,6 +40,28 @@ class ParticipationAdmin(admin.ModelAdmin):
     ]
     date_hierarchy = 'event__start_date'
 
+    @staticmethod
+    def user_email(obj):
+        return obj.user.email
+
+    @staticmethod
+    def user_first_name(obj):
+        return obj.user.first_name
+
+    @staticmethod
+    def user_last_name(obj):
+        return obj.user.last_name
+
+    @staticmethod
+    def user_phone(obj):
+        profile = Profile.objects.filter(user__pk=obj.user.pk).first()
+        return profile.phone if profile else ''
+
+    @staticmethod
+    def user_mobile(obj):
```

Comment:
This name of function can have conflicts with an attribute of the same name. 
To be sure it can be good to use the double underscore standard of python :

 - `user__phone`
 - `user__last_name`
 - `user__first_name`
 - ..